indico/modules/events/static/templates/emails/paper_judgment_notification_email.txt

```diff
@@ -0,0 +1,39 @@
+{% extends 'emails/base.txt' %}
+
+{% macro get_comments() %}
+    {% filter dedent -%}
+        Please see the comments below from the reviewing team:
```

Comment:
"Please see below the comments" - are you sure that's correct? It sounds strange